case_study/code_size/optimized/code_size_package/lib/main.dart

```diff
@@ -0,0 +1,104 @@
+import 'dart:convert';
+import 'dart:typed_data';
+
+import 'package:flutter/material.dart';
+import 'package:steel_crypt/steel_crypt.dart';
```

Comment:
can we import this `as encrypt` so it is clear where we are using the package?

Reply:
Done